third_party/types/gcp-metadata-types.ts

```diff
@@ -0,0 +1,32 @@
+/**
```

Comment:
Is using hyphens in the filenames standard for Node.js?

Reply:
Hyphens commonly used in filenames for cloud-debug-nodejs (where I borrowed the type definitions from) and cloud-trace-nodejs. 
It seems to be a standard practice.